data/journal_check/bug_refs.json

```
@@ -728,5 +728,13 @@
             "sle": ["15-SP6", "15-SP7"]
         },
         "type": "ignore"
+    },
+    "boo#1227081": {
+        "description": "Activation request for 'org.freedesktop.nm_dispatcher' failed.",
```

Regex should be escaped:

```suggestion
        "description": "Activation request for 'org\.freedesktop\.nm_dispatcher' failed\.",
```

---

Pushed a version with this fixed, thanks